index.bs

```diff
@@ -535,101 +539,233 @@ dictionary WriteParams {
   (BufferSource or Blob or USVString)? data;
 };
 
+typedef (BufferSource or Blob or USVString or WriteParams) FileSystemWriteChunkType;
+
 [Exposed=(Window,Worker), SecureContext]
 interface FileSystemWritableFileStream : WritableStream {
-  Promise<void> write((BufferSource or Blob or USVString or WriteParams) data);
+  Promise<void> write(FileSystemWriteChunkType data);
```

Comment:
Hmm. If specs are going to be subclassing us to add convenience methods like this anyway, I wonder if we should add this to WritableStream.prototype...

Alternately this could just be a convenient wrapper for the `{ type: "write" }` variant, and not accept all the different chunk types.

Reply:
Yeah, not sure what the best long-term shape for this API is; this is what we've currently implemented, but I'd be totally up for changes before we ship this outside of an origin trial if we think a different API makes more sense.